src/spec/UtilSpec.ts

```diff
@@ -16,4 +16,10 @@ describe('Utility functions', () => {
       expect(ex.Util.getSideFromVector(ex.Vector.Up)).toBe(ex.Side.Top);
       expect(ex.Util.getSideFromVector(ex.Vector.Down)).toBe(ex.Side.Bottom);
    });
+
+   it('can clamp a number to a maximum and minimum', () => {
+      expect(ex.Util.clamp(0, 10, 20)).toBe(10);
+      expect(ex.Util.clamp(15, 10, 20)).toBe(15);
+      expect(ex.Util.clamp(30, 10, 20)).toBe(20);
```

Comment:
Could you add another couple tests for Infinity and -Infinity to test those boundaries?